routes/api/v1/backend.php

```diff
@@ -44,3 +45,7 @@
 
 // AcademicYear
 Route::apiResource('academicyears', AcademicYearController::class)->only(['index', 'store', 'update', 'destroy']);
+
+// Blood donation request
+Route::apiResource('bloodrequests', BloodDonationRequestController::class)->only(['index','show', 'store', 'update', 'destroy']);
```

Comment:
URL should be like this - `blood-requests`

Reply:
fixed